### ISSUE TITLE
Check for IE 9 is not reliable

### ISSUE THREAD

**Issue by ChristianUlbrich** [taye/interact.js#392]
Great library, however the current check for IE9 is broken, whenever someone is polyfilling `window.atob` in IE9 :)

in [https://github.com/taye/interact.js/blob/master/src/utils/browser.js#L24](https://github.com/taye/interact.js/blob/master/src/utils/browser.js#L24)
```
isIe9OrOlder: domObjects.document.all && !win.window.atob
```
`isIe9OrOlder` is only true, when `win.window.atob` is _undefined_ (or _null_ by the way) - which is true for a stock IE9. However, when someone is poly-filling window.atob - for poor IE9, `isIe9OrOlder` will be false (as has happened in our project). It will also happen, when someone decides to use atob as a global, however I would let burn those poor wretched souls in hell! 

The bottom line is, that you should not rely on existance of global functions for determining a browser version; because they might get polyfilled. I'd suggest a good old user agent regex magic...

**Comment by taye** [comment]
I believe you.
